drivers/iio/light/apds9960.c

```diff
@@ -301,6 +314,7 @@ static const struct iio_chan_spec apds9960_channels[] = {
 		.address = APDS9960_REG_ALS_CHANNEL(CLEAR),
 		.modified = 1,
 		.scan_index = -1,
+		.ext_info = adps9960_ext_info,
```

Comment:
Those should be two different commits: One for adding the `near_level` attribute and one for adding the `MSHW0184` match (which you've already submitted upstream).